routemaster/services/scheduler.rb

```diff
@@ -1,34 +0,0 @@
-require 'rufus-scheduler'
-require 'routemaster/services/scheduler'
-require 'routemaster/services/deliver_metric'
-
-include Routemaster::Services::DeliverMetric
-
-if ENV.fetch('SCHEDULER_ENABLED', false)
-  puts "SCHEDULER_ENABLED"
-  scheduler = Rufus::Scheduler.new
-
-  scheduler.every '5s' do
-    tags = [
-      "env:#{ENV['RACK_ENV']}",
-      'app:routemaster'
-    ]
-
-    Routemaster::Models::Subscription.each do |subscription|
-      Routemaster::Services::DeliverMetric.deliver(
```

Comment:
`deliver` could not be called on the module, I guess that this was the broken part, right?